eve/main.yml

```diff
@@ -927,6 +928,131 @@ stages:
             TF_VAR_rhsm_password: "%(secret:rhel_ci_password)s"
           workdir: build/eve/workers/openstack-single-node-rhel/terraform
 
+  single-node-patch-version:
+    worker: *single_node_worker
+    steps:
+      - SetPropertyFromCommand:
+          name: Set previous patch version to upgrade from and downgrade to
+          property: product_version_prev_patch
+          command: >
+              major=$(echo "%(prop:product_version)s" | cut -d'.' -f1) &&
+              minor=$(echo "%(prop:product_version)s" | cut -d'.' -f2) &&
+              patch=$(echo "%(prop:product_version)s" | cut -d'.' -f3) &&
+              echo "$major.$minor.$(( patch>0 ? patch-1 : 0 ))"
+      - SetPropertyFromCommand:
+          name: Set the patch version flag
+          property: patch_present
+          env:
+            PRODUCT_VERSION: "%(prop:product_version)s"
+            PRODUCT_VERSION_PREV_patch: "%(prop:product_version_prev_patch)s"
+          command: |-
+              if [ "$PRODUCT_VERSION" != "$PRODUCT_VERSION_PREV_patch" ]; then
+                 echo "true"
+              else
+                echo "false"
```

Comment:
detail: indentation is not consistent here

Reply:
merde!